### ISSUE TITLE
Crash when start the game without an internet connection

### ISSUE THREAD

**Issue by sonalder-darlene** [NimaKhajehpour/GuessThatPokemon#15]
When I open the app I click on "Start Game", then choose a language and a generation, when I click on "Start Game" the app crashes.

I tried multiple languages and generation but it always crashes.
Running the GitHub release on GrapheneOS (Android 13) on a Pixel 6A.

**Comment by NimaKhajehpour** [comment]
Is your internet on when you stsrt the game? The game needs intenet otherwise it crashed which i will fix that issue and handle that in next update

**Comment by sonalder-darlene** [comment]
Oh, I've disabled the network permission of the app, I activated it and now it works ^^

**Comment by NimaKhajehpour** [comment]
Yeah the app just pulls data from pokeapi to show details and nothing more so i can assure you that i dont do anything through my app over the internet and its completely safe 😉
But thanks for reminding me of the issue on crashing when internet is off which is not an expected behavior from an app. I'll make sure that the app communicates with user if internet ids off.

**Comment by inson1** [comment]
@NimaKhajehpour Can't you cache it and store it on the phone?
